examples/sentence-embedding/service.py

```diff
@@ -2,18 +2,14 @@
 
 import os
 
-import bentoml_io
 import numpy as np
 import torch
-from pydantic import Field
 
 import bentoml
+from bentoml.ui import Field
```

Comment:
how about `bentoml.io.Field`?

Reply:
But the other members(`File`, `Image` can't be imported from `bentoml.io`)

Reply:
How about `bentoml.Field`?

Reply:
Are we going to also move other IO descriptors to the top namespace?

`bentoml.Image`, `bentoml.File`

Reply:
@ssheng what are we feeling about this?

I'm fine with make this top level.